gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.9-bin.zip
```

Comment:
Double checking: Were there no other changes (such as .gradlew) when you updated?

Reply:
yeah seems wrong - doesnt it download the internet and check it in?

Reply:
the only thing that needs to be updated maybe too is the gradle wrapper itself, but it is not strictly needed always. But we should still try to do it at the same time, and I missed it, this time: see #3675